recruitment/dev-flickr-task.md

```diff
@@ -0,0 +1,30 @@
+# Web Development Task
+
+You've been sent this task as we would love to get an idea of the way you work.
+
+At Holiday Extras we use a variety of different technologies and
+we don't expect you to be aware of all of them! We've designed this task with this in mind. It's worth noting that everything described below is a rough guideline of what we're looking for so don't feel you have to follow it exactly.
+
+Your task is to consume the public [Flickr API](https://api.flickr.com/services/feeds/photos_public.gne?format=json) and display the results according to this basic wireframe:
+
+![mockup](http://i.imgur.com/vyDVR1e.png)
+_[Image source](http://i.imgur.com/vyDVR1e.png)_
```

Comment:
Can we attach the image to this PR and then link it from there? This way we won't be dependent on external sources, it will all be GitHub hosted.